### DISCUSSION TITLE
Maintenance policy

### DISCUSSION THREAD

**Discussion by khbafl** [argoproj/argo-cd#5546] [Q&A] [answered]
Hello, 

What is the maintenance policy of the argocd releases regarding security patches? Is it only the latest version 1.8 that receives security updates? 

For example I am looking for something like this: https://nodejs.org/en/about/releases/

The Argocd documentation does not mention it. 

Thank you,

**Comment by wtam2018** [comment, marked as answer]
Currently,, there is no formal policy in place.   However, you can expect the latest release and the previous one to receive security patches.  E.g. 1.8 (latest) and 1.7 will receive security patches.

**Reply by khbafl** [reply]
Thank you, is that mentionned somewhere on the documentation to use it as official reference? 
Regards,

**Reply by jsoref** [reply]
Not yet. https://github.com/argoproj/argo-cd/issues/5547
